projects/pillow/project.yaml

```diff
@@ -0,0 +1,2 @@
+homepage: "https://python-pillow.org/"
+primary_contact: "guidovranken@gmail.com"
```

Comment:
Would you also like to cc someone at Pillow?

Reply:
I finally got around to this. Please write 1 or more Pillow CC's.

Reply:
security@python-pillow.org thanks

Reply:
Is this address linked to a Google account, or can you do that? If not, you will get e-mail notifications, but cannot see bug report contents.

Reply:
If you'd like this to be merged, then we can at least get the e-mail notifications with that address, and can return to it later if we so choose. Thanks

Reply:
Sure.